release-testing/prepare.sh

```diff
@@ -5,52 +5,88 @@ set -euo pipefail
 . ./executors.sh
 
 function deploy_pg {
-  helm install --namespace "$(store_get TEST_NAMESPACE)" "$(store_get PG_RELEASE_NAME)" \
-   --set postgresqlPassword="$(store_get DB_PASSWORD)" \
+  local TEST_NAMESPACE PG_RELEASE_NAME DB_PASSWORD
+  TEST_NAMESPACE="$(store_get TEST_NAMESPACE)"
+  PG_RELEASE_NAME="$(store_get PG_RELEASE_NAME)"
+  DB_PASSWORD="$(store_get DB_PASSWORD)"
+
+  helm install --namespace "${TEST_NAMESPACE}" "${PG_RELEASE_NAME}" \
+   --set postgresqlPassword="${DB_PASSWORD}" \
    --set persistence.enabled="false" \
    stable/postgresql
 }
 
 function deploy_conjur {
-  helm install --namespace "$(store_get TEST_NAMESPACE)" "$(store_get CONJUR_RELEASE_NAME)" \
-    --set dataKey="$(store_get DATA_KEY)" \
-    --set account="$(store_get CONJUR_ACCOUNT)" \
-    --set image.tag="$(store_get CONJUR_OSS_RELEASE_VERSION)" \
+  local TEST_NAMESPACE
+  local CONJUR_RELEASE_NAME
+  local DATA_KEY
+  local CONJUR_ACCOUNT
+  local CONJUR_OSS_RELEASE_VERSION
+  local AUTHENTICATOR_ID
+  local DB_PASSWORD
+  local PG_RELEASE_NAME
+  local HELM_CHART_RELEASE_VERSION
+
+  TEST_NAMESPACE="$(store_get TEST_NAMESPACE)"
+  CONJUR_RELEASE_NAME="$(store_get CONJUR_RELEASE_NAME)"
+  DATA_KEY="$(store_get DATA_KEY)"
+  CONJUR_ACCOUNT="$(store_get CONJUR_ACCOUNT)"
+  CONJUR_OSS_RELEASE_VERSION="$(store_get CONJUR_OSS_RELEASE_VERSION)"
+  AUTHENTICATOR_ID="$(store_get AUTHENTICATOR_ID)"
+  DB_PASSWORD="$(store_get DB_PASSWORD)"
+  PG_RELEASE_NAME="$(store_get PG_RELEASE_NAME)"
+  HELM_CHART_RELEASE_VERSION="$(store_get HELM_CHART_RELEASE_VERSION)"
+
+  helm install --namespace "${TEST_NAMESPACE}" "${CONJUR_RELEASE_NAME}" \
+    --set dataKey="${DATA_KEY}" \
+    --set account="${CONJUR_ACCOUNT}" \
+    --set image.tag="${CONJUR_OSS_RELEASE_VERSION}" \
     --set image.pullPolicy="Always" \
-    --set authenticators="authn-k8s/$(store_get AUTHENTICATOR_ID)\,authn" \
-    --set databaseUrl="postgres://postgres:$(store_get DB_PASSWORD)@$(store_get PG_RELEASE_NAME)-postgresql.$(store_get TEST_NAMESPACE).svc.cluster.local/postgres" \
-    "https://github.com/cyberark/conjur-oss-helm-chart/releases/download/v$(store_get HELM_CHART_RELEASE_VERSION)/conjur-oss-$(store_get HELM_CHART_RELEASE_VERSION).tgz"
+    --set authenticators="authn-k8s/${AUTHENTICATOR_ID}\,authn" \
+    --set databaseUrl="postgres://postgres:${DB_PASSWORD}@${PG_RELEASE_NAME}-postgresql.${TEST_NAMESPACE}.svc.cluster.local/postgres" \
+    "https://github.com/cyberark/conjur-oss-helm-chart/releases/download/v${HELM_CHART_RELEASE_VERSION}/conjur-oss-${HELM_CHART_RELEASE_VERSION}.tgz"
```

Comment:
Ordering of the `--set` params seems not to follow either importance or alphabetic one